src/server.js

```diff
@@ -100,17 +100,6 @@ function allRequests(req, res, next) {
 	return next();
 }
 
-// Information to populate the About section in UI, either from npm or from git
-var gitCommit = null;
-try {
-	gitCommit = require("child_process")
-		.execSync("git rev-parse --short HEAD 2> /dev/null") // Returns hash of current commit
-		.toString()
-		.trim();
-} catch (e) {
-	// Not a git repository or git is not installed: treat it as npm release
```

Comment:
Yeah, I've heard that before... It's usually followed by "wait, why did I do that already?" a few days/weeks/months/after lol. Actually, [*cough cough*](https://github.com/thelounge/lounge/pull/575/files#r77442290)... 😘

Chop off what?